c/transform.c

```diff
@@ -3,7 +3,19 @@
 
 #include "transform.h"
 
-int outOfChina(double lat, double lng) {
+#undef INLINE
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199900L
+#define INLINE inline
+#else
+#define INLINE
+#endif /* STDC */
+
+#define sqrt(x) __ev_sqrt(x)
+#define fabs(x) __ev_fabs(x)
+INLINE static double __ev_sqrt(x){ return x * x; }
```

Comment:
Very usual, old, black, dark C-world optimization strategy. Doing function calls cost some performance. Therefore, for very trivial functions, people should try to inline as much as possible. For inlining we need to do some *Not Invented Here* or *Wheel-Creating* job (the compiler can't retrieve and inline the implementations in libm). In the past people used straight macros, but inline functions are definitely cleaner.